common/sw-update/dev-updates-profile.cpp

```diff
@@ -47,6 +47,10 @@ namespace rs2
                     if (try_parse_update(_versions_db, _update_profile.device_name, versions_db_manager::RECOMMENDED, comp, recommened_update))
                     {
                         versions_vec[recommened_update.ver] = recommened_update;
+                        if (current_version < recommened_update.ver)
+                        {
+                            recommended_version = *versions_vec.find(recommened_update.ver);
```

Comment:
Why do you need to find it again, if you just assigned it? (line 50)

Reply:
Removed